learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -79,23 +107,14 @@ table<CovidEntry> key(isoCode) covidEntriesTable = table [
     {isoCode: "US", country: "USA", cases: 69808350, deaths: 880976, recovered: 43892277, active: 25035097}
 ];
 ```
-Now, as the data source is completed, let's move on to writing the GraphQL service.
-
-## Write the Ballerina service
 
-Since the data used in this guide map to a GraphQL `Object`, you first have to define this object type. In Ballerina, a
-GraphQL output object can be defined using either a service type or a record type.
+## Create the object types
 
-In this guide, the endpoint is going to return the number of cases in thousands. Therefore, a service type is used to
-define the output object type, and inside the service type, each resource function will return the original value
-divided by 1000.
+Since the data map to a GraphQL `Object`, you need to define this object type. In Ballerina, a GraphQL output object can be defined using either a service type or a record type.
```

Comment:
The existing text `Since the data used in this guide map to a GraphQL Object, you first have to define this object type.` makes more sense to me.

Reply:
Updated.